common/src/transaction/ethereum/abi_contract.rs

```diff
@@ -3,9 +3,10 @@
 use crate::node::ethereum::abi::EthAbiToken;
 use crate::EthError;
 use ethers::prelude::abi::{Contract, Token};
-
+use serde::{Deserialize, Serialize};
 /// Ethereum ABI token to ffi bind
-#[derive(Debug, Eq, PartialEq)]
+// enable serde serialization
+#[derive(Debug, Eq, PartialEq, Serialize, Deserialize)]
```

Comment:
is the default automatic serde derive ok? I just worry about those variants that have `Vec<u8>` that the default one will require a json array of numbers which is a bit inconvenient / wasteful, but it seems it's like that in ethabi/ethers anyway